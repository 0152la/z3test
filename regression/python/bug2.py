from z3 import *

b = BitVec('b', 16)
print Extract(12, 2, b).sexpr()
print Extract(12, 2, b)
print SignExt(10, b).sexpr()
print SignExt(10, b)
print ZeroExt(10, b).sexpr()
print ZeroExt(10, b)
print RepeatBitVec(3, b).sexpr()
print RepeatBitVec(3, b)
