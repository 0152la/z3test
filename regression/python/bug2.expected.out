((_ extract 12 2) b)
Extract(12, 2, b)
((_ sign_extend 10) b)
SignExt(10, b)
((_ zero_extend 10) b)
ZeroExt(10, b)
((_ repeat 3) b)
RepeatBitVec(3, b)
